Dockerfile-princexml

```diff
@@ -0,0 +1,27 @@
+FROM q8s.quadrabee.com/enspirit/startback-api-2.7:0.7
+
+USER root
+
+RUN apt-get update
+RUN apt-get install -y gdebi
+
+ENV HOME /home/app
+WORKDIR /home/app
+
+RUN curl https://www.princexml.com/download/prince_13.5-1_debian10_amd64.deb > prince_13.5-1_debian10_amd64.deb
+RUN gdebi --non-interactive prince_13.5-1_debian10_amd64.deb
+
+RUN gem install --no-document \
+    bundler \
+    path
```

Comment:
We should probably merge the 5 RUN commands above as only one (+ an apt-get clean). to reduce the number of layers of the resulting image.

Reply:
Done.